src/Menu.asm

```diff
@@ -136,11 +136,7 @@ scope Menu {
         lw      a0, 0x0004(s0)              // a0 - (int) current value
         jal     String.itoa_                // v0 = (string) current value
         nop
-        lw      a0, 0x0028(sp)              // a0 = address of info()
-        lw      at, 0x0014(a0)              // at = width
```

Comment:
This line would crash PJ64kve. So instead of passing the info struct in a3 so we can calculated urx here, I calculate urx before calling the subroutine and pass it as a3. No idea why, but hey, it's fixed.